srcpkgs/sublime-merge/template

```diff
@@ -29,13 +29,12 @@ post_extract() {
 
 do_install() {
 	vmkdir "usr/bin"
-	vmkdir "usr/share/sublime_merge"
-	cp --preserve=mode -r ${wrksrc}/* "${DESTDIR}/usr/share/sublime_merge"
+	vmkdir "opt/sublime_merge"
+	cp --preserve=mode -r ${wrksrc}/* "${DESTDIR}/opt/sublime_merge"
```

Comment:
Shouldn't this just use `vcopy`? Also, you're already in `$wrksrc` for `do_install`, no need to use absolute paths.
```
vcopy "*" opt/sublime_merge
```